activerecord/test/cases/relation/or_test.rb

```diff
@@ -180,14 +181,14 @@ def test_structurally_incompatible_values
   # https://www.sqlite.org/limits.html#max_expr_depth
   class TooManyOrTest < ActiveRecord::TestCase
     unless current_adapter?(:SQLite3Adapter)
-      fixtures :citations
```

Comment:
Loads 65k records into the database. I believe a smaller amount in this case would suffice.